tools/flaky-test-retryer/github_commenter.go

```diff
@@ -118,30 +123,40 @@ func (gc *GithubClient) getOldComment(org, repo string, pull int) (*github.Issue
 
 // parseEntries collects retry information from the given comment, so we can reuse it in
 // a new comment.
-func parseEntries(comment *github.IssueComment) (map[string]int, error) {
-	entries := map[string]int{}
+func parseEntries(comment *github.IssueComment) (map[string]*entry, error) {
+	entries := make(map[string]*entry)
 	if comment == nil {
 		return entries, nil
 	}
 	re := regexp.MustCompile(`.* \| \d`)
 	entryStrings := re.FindAll([]byte(comment.GetBody()), -1)
 	for _, e := range entryStrings {
 		fields := strings.Split(string(e), " | ")
-		retry, err := strconv.Atoi(strings.Split(fields[1], "/")[0])
+		// support old comments with 2 columns, before links were added
+		var links string
+		if len(fields) == 2 {
+			links = ""
+		} else {
+			links = fields[1]
+		}
+		retry, err := strconv.Atoi(strings.Split(fields[len(fields)-1], "/")[0])
```

Comment:
We are always assuming that the comment here will be well-formed. We should confirm that len(fields) == 3 atleast before using this

Reply:
done